_meta/mozilla-mobile-android-components.json

```diff
@@ -0,0 +1,6 @@
+{
+  "name": "mozilla-mobile/android-components",
+  "revs": {
+    "master": "7ad0aa0c6dad45346ba156061afecd22422307c5"
```

Comment:
Is this use to track the changeset conversion from the original repository?

Reply:
Yes, that's what's indicating the version for each repository. Each in a separate file so that we don't have conflicts between quarantine branches. That's what we talked about last week when I had the pontoon commits and branches and stuff on the whiteboard.